requirements/constraints.txt

```diff
@@ -7,4 +7,3 @@
 # link to other information that will help people in the future to remove the
 # pin when possible.  Writing an issue against the offending project and
 # linking to it here is good.
-edx-codejail==3.2.0
```

Comment:
This has never happened, but what if a new build for the Redwood image is made after a breaking version of edx-codejail is released?

Reply:
The `tutor-contrib-codejail` plugin [targets the `release/redwood.1`](https://github.com/eduNEXT/tutor-contrib-codejail/blob/main/tutorcodejail/plugin.py#L33) tag of this repository in redwood, so any build using the `V18` of the plugin has this constraint.

This actually isn't the same version used by edx-platform on redwood, so I'm open to bumping edx-codejail to 3.4.1 and making a new redwood tag for this repo.

Reply:
Oh right! I completely forgot, thank you. Considering the stability of edx-codejail, would it be worth pinning only for the test environment?

Reply:
I think the tests are fine using the available `.txt`s, they would only change when we manually run the requirements upgrade, which should be each time we are preparing a new release.